File: app/src/main/java/de/christianost/aida/activities/StartActivity.java
package de.christianost.aida.activities;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import de.christianost.aida.R;

public class StartActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start);
    }

    /**
     * Handles clicking the "Create New Sketch" button.
     *
     * @param view
     */
    public void onCreateNewSketch(View view) {
        Intent intent = new Intent(this, SketchActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/kotlin/de/christianost/aida/activities/StartActivity.kt
package de.christianost.aida.activities

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

import de.christianost.aida.R

class StartActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_start)
    }

    /**
     * Handles clicking the "Create New ProcessingSketch" button.
     *
     * @param view
     */
    fun onCreateNewSketch(view: View) {
        val intent = Intent(this, SketchActivity::class.java)
        startActivity(intent)
    }
}
